tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -227,6 +227,7 @@ class QualificationAppInfo(
 
   private[qualification] def processSQLPlan(sqlID: Long, planInfo: SparkPlanInfo): Unit = {
     checkMetadataForReadSchema(sqlID, planInfo)
+    checkJdbcScan(sqlID, planInfo)
```

Comment:
seems like we could easily have this apply to profiling tool but adding this same line into the ApplicationInfo class and updating tests and docs.